src/assets/scss/_variables.scss

```diff
@@ -0,0 +1,43 @@
+$primary: #f15151;
+$error: #ef5777;
+$success: #1abc9c;
+$light: #f5f8fa;
+$medium: #657786;
+$dark: #34495e;
+$white: #fff;
+$black: #000;
+
+$offWhite: #f8f8f8;
+$medGray: #858585;
+$darkGray: #3C3C3C;
+
+$emGreen: #1AA9A5;
```

Comment:
sorry Yuxuan & I made this color darker to pass contrast ratio tests but forgot to update to Figma, it should be #148481